web/src/features/map/Map.tsx

```diff
@@ -104,25 +109,25 @@ export default function MapPage({ onMapLoad }: MapPageProps): ReactElement {
     [theme]
   );
 
-  const { isLoading, isSuccess, isError, data } = useGetState();
+  const { isLoading, isError, data } = useGetState();
   const mapReference = useRef<MapRef>(null);
   const { worldGeometries, statesGeometries } = useGetGeometries();
+  const map = mapReference.current?.getMap();
+  map?.touchZoomRotate.disableRotation();
+  map?.touchPitch.disable();
```

Comment:
Doesn't this part need to be within a useEffect so that it runs when mapReference changed? Is it always available immediately upon rendering Map?

Reply:
I think the map reference should stay the same but I've moved this to inside a useEffect so it's not called every re-render